pennylane/tape/tape.py

```diff
@@ -443,6 +428,8 @@ def stop_recording(self):
     # construction methods
     # ========================================================
 
+    _queue_category = "_ops"
```

Comment:
Is this required because tapes can be queued inside other tapes

Reply:
added comment to that effect.